src/test/java/DayThreeTest.java

```diff
@@ -24,12 +24,14 @@ public void opCode2MultipliesPostion1And2Together(){
     public void opCode99ImmediatelyHaltsProgramWithMoreThan4Numbers(){
         Assertions.assertThat(dayThree.opcodeProgram("2,4,4,5,99,0")).contains("2, 4, 4, 5, 99, 9801");
     }
+    //what's the difference between this test and the previous one?
```

Comment:
Seems to check the same thing and has two difference assertions, which makes me suspicious.